uvtools/dspec.py

```diff
@@ -775,3 +868,71 @@ def delay_filter_leastsq(data, flags, sigma, nmax, add_noise=False,
         mdl_array[i, :] = bf_model
 
     return mdl_array, cn_array, inp_data
+
+
+def sinc_downweight_mat_inv(nchan, df, filter_centers, filter_widths,
+                            filter_factors, cache = {}, wrap = False, wrap_interval=1,
+                            nwraps = 1000, no_regularization = False):
+    """
+    Computes inverse of clean weights for a baseline.
```

Comment:
did we say we weren't referencing "clean", and is this a holdout of that? I forget

Reply:
you're right. I'm removing the reference.